### PR TITLE
Added unsupportedClasses to Fixture

### PR DESCRIPTION
**Pull request by segabriel**
Motivation: currently I can't ask for `TestInfo` instance as an argument of JUnit methods.
So as an option will be nice to have a method which will specify which classes should be excluded for `proxyFor` method.

### CODE REVIEW

**Comment by ronenhamias** [comment]
can you provide example where this is useful and why?

**Comment by segabriel** [comment]
@ronenhamias 
In exchange tests with the Fixture on Travis, some test was failed and had no chance to decide which one exactly failed. So I wanted to add test separator via logging like this (https://github.com/scalecube/scalecube-services/blob/d63089316d20ba4dcf87494b3d18462a9f0271ff/services/src/test/java/io/scalecube/services/BaseTest.java) 
But unfortunately, it throws an exception (can be more than one `ParameterResolver` which supports necessary type).

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixtures.java [hunk 1]
```diff
@@ -110,7 +110,11 @@ public boolean supportsTestTemplate(ExtensionContext context) {
   public boolean supportsParameter(
       ParameterContext parameterContext, ExtensionContext extensionContext)
       throws ParameterResolutionException {
-    return getStore(extensionContext).map(getFixtureFromStore).isPresent();
+    Class<?> type = parameterContext.getParameter().getType();
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixtures.java]:
```suggestion
    Class<?> type = parameterContext.getParameter().getType();
        if (type
        .getPackage()
        .getName()
        .startsWith("org.junit.jupiter.api")) {
      return false;
    }
```

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixtures.java [hunk 2]
```diff
@@ -110,7 +110,11 @@ public boolean supportsTestTemplate(ExtensionContext context) {
...
+    return getStore(extensionContext)
+        .map(getFixtureFromStore)
+        .map(fixture -> fixture.unsupportedClasses().stream().noneMatch(type::equals))
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixtures.java]:
```suggestion
        .isPresent();
```

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixtures.java [hunk 3]
```diff
@@ -110,7 +110,11 @@ public boolean supportsTestTemplate(ExtensionContext context) {
...
+        .orElse(false);
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixtures.java]:
```suggestion
```

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java [hunk 4]
```diff
@@ -84,4 +90,8 @@
   default String name() {
     return getClass().getSimpleName();
   }
+
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java]:
```suggestion
```

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java [hunk 5]
```diff
@@ -84,4 +90,8 @@
...
+  default Collection<Class<?>> unsupportedClasses() {
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java]:
```suggestion
```

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java [hunk 6]
```diff
@@ -84,4 +90,8 @@
...
+    return Arrays.asList(RepetitionInfo.class, TestInfo.class, TestReporter.class);
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java]:
```suggestion
```

File: scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java [hunk 7]
```diff
@@ -84,4 +90,8 @@
...
+  }
```
Comment (aharonha) [inline scalecube-test-utils/src/main/java/io/scalecube/test/fixtures/Fixture.java]:
```suggestion
```

**Pull request merged**